src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlRetry/SqlRetryService.cs

```diff
@@ -464,6 +468,7 @@ public async Task<SqlConnection> GetConnection(ISqlConnectionBuilder sqlConnecti
                 // must be set before opening connection to take effect. Therefore we must reset it to null here before opening the connection.
                 conn.RetryLogicProvider = null; // To remove this line _sqlConnectionBuilder in healthcare-shared-components must be modified.
                 await conn.OpenAsync(cancel);
+                logger.LogInformation("Opened connection to the database in {ConnectionOpenSeconds} seconds.", sw.Elapsed.TotalSeconds);
```

Comment:
Could this include if the connection is to a read or write replica? I know it doesn't have much use now, but it could be helpful in the future.

Reply:
Added